chain/consensus/mir/manager.go

```diff
@@ -155,6 +155,8 @@ func NewManager(ctx context.Context,
 		initialValidatorSet: initialValidatorSet,
 		membership:          membership,
 	}
+	m.mirErrChan = make(chan error, 1)
+	m.mirCtx, m.mirCancel = context.WithCancel(context.Background())
```

Comment:
Can we add a comment here to describe the reason for starting this context here?

Reply:
Ok, I see that you describe it below, so no need